src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml

```diff
@@ -1,6 +1,91 @@
 admin_country_states:
   path: /
```

Comment:
Can we avoid the URL duplication with `admin_states_index` ? I know there is the `request.isXmlHttpRequest()` condition but it could be confusing with junior developers struggling with Symfony.

We can pick something very unambiguous like `/country-states`

Reply:
Or put the `id_country` argument into path : `/country/{countryId}/states` ; `/country/{countryId}/all` ; `/country-data/{countryId}/`